subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/RealLifeAndroidBuildPerformanceTest.groovy

```diff
@@ -155,7 +155,8 @@ class RealLifeAndroidBuildPerformanceTest extends AbstractCrossVersionPerformanc
             "--add-opens",
             "java.base/java.net=ALL-UNNAMED"
         ]) // needed when tests are being run with CC on, see https://github.com/gradle/gradle/issues/22765
-        runner.addBuildMutator {is -> new AgpAndKgpVersionMutator(is, agpVersion, kgpVersion) }
+        runner.addBuildMutator { is -> new SupplementaryRepositoriesMutator(is) }
```

Comment:
🤔 Why we're going through obscure ways to change the build file instead of simply changing it?